blurring_as_a_service/training_pipeline/submit_training_pipeline.py

```diff
@@ -0,0 +1,52 @@
+from azure.ai.ml import Input, Output
+from azure.ai.ml.constants import AssetTypes
+from azure.ai.ml.dsl import pipeline
+
+from blurring_as_a_service.settings.settings import BlurringAsAServiceSettings
+from blurring_as_a_service.training_pipeline.components.train_model import train_model
+from blurring_as_a_service.utils.aml_interface import AMLInterface
+
+
+@pipeline()
+def training_pipeline(training_data):
+    train_model_step = train_model(mounted_dataset=training_data)
+    train_model_step.outputs.yolo_yaml_path = Output(
+        type="uri_folder", mode="rw_mount", path=training_data.path
+    )
+    return {}
+
+
+def main():
+    aml_interface = AMLInterface()
+    settings = BlurringAsAServiceSettings.get_settings()
+
+    # TODO: Impossible to create an env for the training with the SDKv2 at the moment.
```

Comment:
Do we skip creating the environment here? 
I assume we use one that already exists and works

Reply:
Yes, indeed. And I think that the environment creation should be a stand-alone job to run when a new env is needed. What do you think?

Reply:
Alright, so we just throw an error if the env does not exist I suppose

Reply:
Yess, that's automatically handled by the SDK.